webapp/components/Modal/ReportModal.vue

```diff
@@ -9,13 +9,35 @@
     <!-- eslint-disable-next-line vue/no-v-html -->
     <p v-html="message" />
 
+    <ds-radio
+      :value="form.reasonCategory"
+      :schema="formSchema.reasonCategory"
+      :label="$t('report.reason.category.label')"
+      :options="form.reasonCategoryOptions"
+      labelProp="label"
+    />
+    <ds-input
+      class="reason-description"
+      :value="form.reasonDescription"
+      :schema="formSchema.reasonDescription"
+      :label="$t('report.reason.description.label')"
+      :placeholder="$t('report.reason.description.placeholder')"
+      type="textarea"
+      rows="5"
+    />
+    <small class="smallTag">
+      {{ form.reasonDescription.length }}/{{ formSchema.reasonDescription.max }}
+    </small>
+    <ds-space />
+
     <template slot="footer">
       <ds-button class="cancel" icon="close" @click="cancel">{{ $t('report.cancel') }}</ds-button>
 
       <ds-button
         danger
         class="confirm"
         icon="exclamation-circle"
+        :disabled="failsValidations"
```

Comment:
in this case, difficult to achieve because we have a `<template #footer>` wrapping the `cancel/report` buttons... you cannot use `<template slot-scope="{ errors }">` with nested `templates`, but if we convert this to a `<div slot="footer">`, it completely changes the `styling` of the buttons to a `green` confirm button, instead of the desired `red` report button